.github/workflows/ci-release.yml

```diff
@@ -40,7 +40,7 @@ jobs:
   # If this was a workflow dispatch event, we need to generate and push a tag
   # for goreleaser to grab
   version_bump:
-    needs: [lint, test]
+    needs: [lint, test, goreleaser-check]
```

Comment:
hmm this seems to be missing the definition for `goreleaser-check`

Reply:
Resolved by backporting https://github.com/celestiaorg/celestia-app/pull/2652 in this PR